compio-buf/src/io_buf.rs

```diff
@@ -291,14 +295,18 @@ pub unsafe trait IoBufMut: IoBuf + SetBufInit {
         }
     }
 
+    /// Create an [`IoSliceMut`] of the uninitialized part of the buffer.
+    fn as_io_slice_mut(&mut self) -> IoSliceMut {
+        IoSliceMut::new(unsafe { std::mem::transmute(self.as_uninit_slice()) })
```

Comment:
Transmuting MaybeUninit to init u8 is UB. I think we do need our own io slice type that supports MaybeUninit.

Reply:
I'm adding custom `IoSlice` in #94

Reply:
Fixed